queries/highlights.scm

```diff
@@ -9,12 +9,10 @@
   (null)
   (na)
 ] @constant.builtin
-(string) @string
 
-(escape_sequence) @constant.character.escape
+((string) @string (#match? @string "^\".+?\""))
```

Comment:
```suggestion
(string) @string
```

All strings should be accounted for, both quoted and unquoted.

Reply:
If we do that, the all text will be in string style "green", That means there is no highlight diffenrence.

Reply:
<img width="511" alt="image" src="https://github.com/arnau/tree-sitter-csv/assets/5518/ce88dfa1-1e49-4a08-b300-a9e35066459b">

Reply:
correct, both are strings so I don't expect there to be any difference. By preserving `(quote) @punctuation.bracket` you'll have quotes in a different colour so they'll serve as a visual marker (assuming the colour theme does not force the same colour of course).